assets/css/style.css

```diff
@@ -4,302 +4,339 @@ body {
    margin: 0;
    width: 100%;
    box-sizing: border-box;
+   font-family: "Poppins"
 }
 
-/* Poppins */
-@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400;1,500&display=swap');
-/* Ubuntu */
-@import url('https://fonts.googleapis.com/css2?family=Exo+2:wght@700&family=Ubuntu:wght@300;400&display=swap');
-/* Exo 2 */
-@import url('https://fonts.googleapis.com/css2?family=Exo+2:wght@400;700&display=swap');
 
 /* Navbar CSS Start*/
 .navbar {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    background-color: #f445;
-    color: #333;
-    padding: 10px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
-  }
-  
-  .navbar-logo {
-    font-size: 24px;
-    font-weight: bold;
-  }
-  
-  .navbar-nav {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    list-style: none;
-    margin: 0;
-    padding: 0;
-  }
-  
-  .nav-item {
-    margin-right: 20px;
-  }
-  
-  .nav-link {
-    color: #333;
-    text-decoration: none;
-    font-size: 18px;
-    font-weight: bold;
-    transition: color 0.3s ease;
-  }
-  
-  .nav-link:hover {
-    color: #f00;
-  }
-  
-  .navbar-toggle {
-    display: none;
-    cursor: pointer;
-  }
-  
-  .navbar-toggle span {
-    display: block;
-    width: 25px;
-    height: 3px;
-    background-color: #333;
-    margin: 5px;
-    transition: transform 0.3s ease;
-  }
-  
-  @media (max-width: 768px) {
-    .navbar-nav {
-      display: none;
-      position: absolute;
-      top: 60px;
-      left: 0;
-      right: 0;
-      background-color: #fff;
-      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
-      padding: 10px;
-      flex-direction: column;
-      justify-content: center;
-      align-items: center;
-    }
-  
-    .nav-item {
-      margin: 10px 0;
-    }
-  
-    .navbar-toggle {
-      display: block;
-    }
-  
-    .navbar-toggle.active span:nth-child(1) {
-      transform: rotate(45deg) translate(5px, 5px);
-    }
-  
-    .navbar-toggle.active span:nth-child(2) {
-      opacity: 0;
-    }
-  
-    .navbar-toggle.active span:nth-child(3) {
-      transform: rotate(-45deg) translate(5px, -5px);
-    }
-  }
-  /* End Navbar CSS  */
+   background-color: rgb(255, 255, 255);
+   color: #333;
+   padding: 10px;
+   width: 100vw;
+}
+
+.navbar nav {
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   list-style: none;
+}
+
+.nav-link ul {
+   display: flex;
+   list-style: none;
+}
+
+.nav-link ul li a {
+   color: #333;
+   text-decoration: none;
+   padding-left: 1vw;
+   padding-right: 1vw;
+}
+
+.nav-link ul li a:hover {
+   color: #670B78;
+}
+
```

Comment:
Kindly add
 .nav-item{
   margin-right: 5%;
}
here

Reply:
Ok

Reply:
Oh, 
I noticed you had accidentally omitted some code. Kindly push the first changes and we'll go through the rest of the code